health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Layout.js

```
@@ -0,0 +1,184 @@
+        );
+      case "heatmap":
+        //TODO: Add HeatMap chart back
+        return <div>HeatMap has to rendered here</div>
+        // return <HeatMapChart visualizer={chart} chartId={chart?.id} isNational={false} showLabel={true} pageZoom={pageZoom} />;
+      case "bar":
+      case "sideBySideBar":
+        return <CustomHorizontalBarChart data={chart} title={title} yAxisLabel={showCustomLabel(title, t)} pageZoom={pageZoom} downloadChartsId={downloadChartsId} />;
+      case "venn":
+        return <VennDiagram data={chart}/>;
+      case "kibanaComponent":
+        return <KibanaCard moduleName={chart?.moduleName} pageName={chart?.pageName} filters={value}></KibanaCard>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)